test/planned_subvol_test.rb

```diff
@@ -207,4 +208,122 @@
     end
   end
 
+  context ".read_from_xml" do
```

Comment:
Sorry, I was not aware of this pull request and I already added tests for reading from xml in this other PR that fix a problem in that part (duplicated subvolumes).